### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by rrusso** [lsuits/tues#1]
What about LSU OES?

the LSU Open Enrollment System?

**Comment by chadmazilly** [comment]
Yes I like that. I knew that TUES was a little cheeky, but it seemed to fit in the moment. How about we call it LSU OES, enrol_oes?

**Comment by rrusso** [comment]
I like the idea of Open being part of it.
